libs/agno/agno/agent/agent.py

```diff
@@ -4134,7 +4153,35 @@ def _create_run_data(self) -> Dict[str, Any]:
 
         return run_data
 
-    def _log_agent_run(self, session_id: str, user_id: Optional[str] = None) -> None:
+    def _register_agent_on_platform(self) -> None:
```

Comment:
```suggestion
    def register_agent_on_platform(self) -> None:
```
Since it is being called outside of the agent.